networkit/test/test_independentset.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import unittest
+import networkit as nk
+
+class TestGraphTools(unittest.TestCase):
+	def testLubyAlgorithm(self):
+		G = nk.Graph(4, False, False)
+		G.addEdge(0, 1)
+		G.addEdge(0, 2)
+		G.addEdge(1, 2)
+		G.addEdge(2, 3)
+
+		luby = nk.independentset.Luby()
+		res = luby.run(G)
+		count = 0
+		for id in res:
+			if id == True:
+				count = count +1
```

Comment:
A simpler version could be:
```suggestion
		count = sum(res)
```